src/components/EditablePermissions/EditablePermissions.js

```diff
@@ -164,7 +169,7 @@ class EditablePermissions extends React.Component {
           <DropdownMenu
             data-role="menu"
             width="40em"
-            aria-label={<FormattedMessage id="ui-users.permissions.availablePermissions" />}
+            aria-label={this.props.intl.formatMessage({ id: 'ui-users.permissions.availablePermissions' })}
```

Comment:
@zburke As we already destructure props above, why not to add intl also to destructuring.